sys/net/routing/rpl/of0.c

```diff
@@ -78,7 +78,7 @@ uint16_t calc_rank(rpl_parent_t *parent, uint16_t base_rank)
 /* We simply return the Parent with lower rank */
 rpl_parent_t *which_parent(rpl_parent_t *p1, rpl_parent_t *p2)
 {
-    if (p1->rank < p2->rank) {
+    if (p1->rank <= p2->rank) {
```

Comment:
I see. Then I would vote to add a comment explaining which parent is returned for equal ranks and a best practice to use it.